catalyst-gateway/bin/Cargo.toml

```diff
@@ -18,6 +18,8 @@ workspace = true
 cardano-chain-follower = {version = "0.0.6", git = "https://github.com/input-output-hk/catalyst-libs.git", tag="v0.0.10" }
 c509-certificate = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "v0.0.3" }
 rbac-registration = { version = "0.0.2", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "v0.0.8" }
+catalyst-signed-doc = { version = "0.0.1", git = "https://github.com/input-output-hk/catalyst-libs.git", branch = "fix/replace-ulid-with-uuidv7" }
```

Comment:
This branch needs to be merged to main, and tagged in cat-libs before we can merge this PR to main.